feat: add update 82 (#122)
tari-project/tari-dot-com: _updates/2022-07-08-update-82.md

---
layout: update
tag: Developer Update
date: 2022-07-08
author: stringhandler
thumbnail: update-background.jpg
title: This week in Tari
class: subpage
---

> Tari's DAN Layer is progressing nicely with a lot of pull requests (PR's) related to checkpoints and committees getting merged this week. 

Reviewer: ```suggestion
Tari's DAN Layer is progressing nicely with a lot of pull requests (PRs) related to checkpoints and committees getting merged this week. 
```

> One exciting PR merged this week was **[#4260](https://github.com/tari-project/tari/issues/4260) Add range proof batch verification to validators**. More than 80 percent of CPU time during block sync and verification is taken up by range proof verification. Using a clever trick of using the same cryptographic generators, multiple rangeproofs can be verified at the same time, dramatically cut down the time taken in verification.

Reviewer: ```suggestion
One exciting PR merged this week was **[#4260](https://github.com/tari-project/tari/issues/4260) Add range proof batch verification to validators**. More than 80 percent of CPU time during block sync and verification is taken up by range proof verification. Using a clever trick of using the same cryptographic generators, multiple range proofs can be verified at the same time, dramatically cutting down the time taken in verification.
```

> 

Reviewer 2: Talk about leaving us hanging! Where's the benchmarks showing the speed-up?

> Here's a list of other PR's recently merged:

Reviewer: ```suggestion
Here's a list of other PRs recently merged:
```
